FAQ entry: Role of pods (#1100)
wandb/docs: content/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

## FAQ

### What is the purpose/role of each individual pod?
* **`wandb-app`**: the core of W&B, including the GraphQL API and frontend application. It powers most of our platform’s functionality.
* **`wandb-console`**: the administration console, accessed via `/console`. 
* **`wandb-otel`**: the OpenTelemetry agent, which collects metrics and logs from resources at the Kubernetes layer for display in the administration console.
* **`wandb-prometheus`**: the Prometheus server, which captures metrics from various components for display in the administration console.

> * **`wandb-parquet`**: a backend microservice separate from the wandb-app pod, responsible for exporting database data to object storage in Parquet format.

Reviewer: ```suggestion
* **`wandb-parquet`**: a backend microservice separate from the `wandb-app` pod that exports database data to object storage in Parquet format.
```

* **`wandb-weave`**: another backend microservice that loads query tables in the UI and supports various core app features.

> * **`wandb-weave-trace`**: a framework for tracking, experimenting with, evaluating, deploying, and improving LLM-based applications. The framework is accessed via wandb-app.

Reviewer: ```suggestion
* **`wandb-weave-trace`**: a framework for tracking, experimenting with, evaluating, deploying, and improving LLM-based applications. The framework is accessed via the `wandb-app` pod.
```

## FAQ

> ### What is the purpose/role of the individual pods?

Reviewer: ```suggestion
### What is the purpose/role of each individual pod?
```

> * **wandb-app:** The core of our backend, including the GraphQL API and frontend application. It powers most of our platform’s functionality.
> * **wandb-console:** The administration console is a separate application and can be accessed via /console. 
> * **wandb-otel:** OpenTelemetry collects metrics and logs from Kubernetes resources to be displayed in the administration console.
> * **wandb-prometheus:** Prometheus captures metrics from various components to be displayed in the administration console.
> * **wandb-parquet:** A backend microservice separate from the wandb-app pod, responsible for exporting database data to object storage in Parquet format.
> * **wandb-weave:** Another backend microservice that loads query tables in the UI and supports various core app features.
> * **wandb-weave-trace:** A framework for tracking, experimenting with, evaluating, deploying, and improving LLM-based applications. The framework is accessed via wandb-app.

Reviewer: Please use code font for the strings like `wandb-app`, and please put the colons outside of the bold. :) I've changed the first few to show what I mean. I've also made changes to avoid "our" and "backend", put endpoints in code font, lowercased the "the" right after the term,. and avoided some redundancy. PTAL.
```suggestion
* **`wandb-app`**: the core of W&B, including the GraphQL API and frontend application. It powers most of our platform’s functionality.
* **`wandb-console`**: the administration console, accessed via `/console`. 
* **`wandb-otel`**: the OpenTelemetry agent, which collects metrics and logs from resources at the Kubernetes layer for display in the administration console.
* **`wandb-prometheus`**: the Prometheus agent, which captures metrics from various components for display in the administration console.
* **wandb-parquet:** A backend microservice separate from the wandb-app pod, responsible for exporting database data to object storage in Parquet format.
* **wandb-weave:** Another backend microservice that loads query tables in the UI and supports various core app features.
* **wandb-weave-trace:** A framework for tracking, experimenting with, evaluating, deploying, and improving LLM-based applications. The framework is accessed via wandb-app.
```

> 

Reviewer: Nit
```suggestion
```
